tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,179 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
```

Comment:
@spokerman12, since the goal is to ensure that the existing behaviour without specifying the 'lookup_range' option should continue to work as-is, can you write a test case for it?

Reply:
True! Will do.

Reply:
This is now tested in `test_lookup_range_undefined` 👍🏼